pom.xml

```diff
@@ -1,237 +1,228 @@
 
-	<dependencies>
-		<!-- TDS -->
-		<dependency>
-			<groupId>org.opentestsystem.delivery</groupId>
-			<artifactId>tds-exam-client</artifactId>
-			<version>${tds-exam-client.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.opentestsystem.delivery</groupId>
-			<artifactId>tds-session-client</artifactId>
-			<version>${tds-session-client.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.opentestsystem.delivery</groupId>
-			<artifactId>tds-assessment-client</artifactId>
-			<version>${tds-assessment-client.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.opentestsystem.delivery</groupId>
-			<artifactId>tds-common</artifactId>
-			<version>${tds-common.version}</version>
-		</dependency>
+    <dependencies>
+        <!-- TDS -->
+        <dependency>
+            <groupId>org.opentestsystem.delivery</groupId>
+            <artifactId>tds-exam-client</artifactId>
+            <version>${tds-exam-client.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.opentestsystem.delivery</groupId>
+            <artifactId>tds-session-client</artifactId>
+            <version>${tds-session-client.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.opentestsystem.delivery</groupId>
+            <artifactId>tds-assessment-client</artifactId>
+            <version>${tds-assessment-client.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.opentestsystem.delivery</groupId>
+            <artifactId>tds-common</artifactId>
+            <version>${tds-common.version}</version>
+        </dependency>
 
-		<!-- Utilities -->
-		<dependency>
-			<groupId>org.apache.commons</groupId>
-			<artifactId>commons-lang3</artifactId>
-			<version>3.5</version>
-		</dependency>
+        <!-- Utilities -->
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.5</version>
+        </dependency>
 
-		<!-- Spring -->
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-web</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-configuration-processor</artifactId>
-			<optional>true</optional>
-		</dependency>
-		<dependency>
-			<groupId>org.springframework</groupId>
-			<artifactId>spring-webmvc</artifactId>
-		</dependency>
+        <!-- Spring -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+            <exclusions>
```

Comment:
Excluded tomcat since our applications use undertow.  Added the undertow starter right below this dependency.